README.md

```diff
@@ -1,24 +1,55 @@
-# README
+# BLOG-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Blog app is classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
```

Comment:
- [ ] The project description should be about what the project is about and not the requirements. Kindly update it.